astroid/brain/brain_collections.py

```diff
@@ -77,7 +77,69 @@ def _ordered_dict_mock():
     class OrderedDict(dict):
         def __reversed__(self): return self[::-1]
         def move_to_end(self, key, last=False): pass"""
+    if PY39:
+        base_ordered_dict_class += """
+        @classmethod
+        def __class_getitem__(cls, item): return cls"""
     return base_ordered_dict_class
 
 
-astroid.register_module_extender(astroid.MANAGER, "collections", _collections_transform)
+def _collections_module_properties(node, context=None):
+    """
+    Adds a path to a fictive file as the _collections module is a pure C lib.
+    """
+    node.file = "/tmp/unknown"
+    return node
+
+
+astroid.MANAGER.register_transform(
+    astroid.Module, _collections_module_properties, lambda n: n.name == "_collections"
+)
+astroid.register_module_extender(
+    astroid.MANAGER, "_collections", _collections_transform
+)
+
+
+PY39 = sys.version_info >= (3, 9)
```

Comment:
Minor: This is already at the top of the file. Do we need to define it again?

Reply:
Definitely useless!